src/components/TransactionFlow/displays/FromToAccount.tsx

```diff
@@ -94,7 +94,7 @@ const FromToAccount = ({ networkId, fromAccount, toAccount, displayToAddress = t
           />
         </AddressWrapper>
       </AddressContainer>
-      {displayToAddress && (
+      {displayToAddress && toAccount.address && (
```

Comment:
This should be unnecesary https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/components/TransactionFlow/TxReceipt.tsx#L419

Perhaps the txType was missing causing the issue?

Reply:
There is no `txReceipt` (which is used to get the `txType`) at all it looks like. I'm not sure what a good way to solve that would be?

Reply:
This should be set here?
https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/features/DeployContracts/stateFactory.tsx#L99-L135